wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/service/TextDocumentService.java

```diff
@@ -169,6 +187,13 @@ public void configureMethods() {
         .paramsAsString()
         .resultAsString()
         .withFunction(this::getFileContent);
+
+    requestHandler
+        .newConfiguration()
+        .methodName("textDocument/snippets")
```

Comment:
Move to common constants if possible, please

Reply:
Makes sense when we extract constants from the whole class.